docs/community/2-sdlc/continuous-integration/code-security-guidelines.md

```diff
@@ -198,8 +198,8 @@ enforced across the Alliance's repositories.
       * **Ed-Fi-Model**
       * **Ed-Fi-Standard**
       * **Ed-Fi-TPDM-Artifacts**
-    * **Starter Kits** that do not contain any scannable code or actions:
-      * **Ed-Fi-Starter-Kit-Assessments**
-      * **Ed-Fi-Starter-Kit-Equity**
+    * **Dashboards** that do not contain any scannable code or actions:
+      * **Ed-Fi-Assessments-Dashboard**
+      * **Ed-Fi-Equity-Dashboards**
```

Comment:
I also renamed these repositories in GitHub.

Reply:
I'm not completely happy leaving "Starter Kit Support" there, but the priority for changing it is very low. Therefore, I decided that it is not worth the effort compared to other tasks.